Feature/events html structure & scss-styling #23

leolinder-hash opened the pull request:
This PR refers to card #23 "events/ Create HTML structure for the event-page". 

Included in this feature is:
1) HTML-structure & elements for the event-page
2) Styling with SCSS for mobile (480px+), tablet (768px+) and desktop (900px+)
3) Sections with cards for popular events & for upcoming events
4) A fitting welcome picture to the event-page

NOTICE!
- Some of the colors are preliminary and can be adjusted later
- The header (menu) will be added after it¨s been merged with main

See pictures below for a basic understanding of what the page looks like:

**Mobile:**
<img width="310" height="766" alt="Mobile_1" src="https://github.com/user-attachments/assets/0f09cd64-c56e-44c1-b68c-46e2dc596f10" />

<img width="290" height="760" alt="Mobile_2" src="https://github.com/user-attachments/assets/9b4256d3-1205-43be-9eb2-dcbf089927ec" />

**Tablet**
<img width="991" height="777" alt="tablet_1" src="https://github.com/user-attachments/assets/4da232f4-d0d3-4419-885e-04e795aa70e1" />

<img width="955" height="758" alt="tablet_2" src="https://github.com/user-attachments/assets/834fdf16-3697-434b-affc-c374f92f3fa6" />

**Desktop**
<img width="1117" height="777" alt="desktop_1" src="https://github.com/user-attachments/assets/1462e14d-02d7-4927-b7ed-29c0e0808fd6" />

<img width="1117" height="747" alt="desktop__2" src="https://github.com/user-attachments/assets/59165507-44d0-4bff-bca5-1c0e7c2c35bf" />

ImanBatyah reviewed (approved):
Ser riktigt bra ut 👍
Snygg och tydlig HTML-struktur för event-sidan och bra att du har kollat responsivt (480/768/900+).
Gillar också att du återanvänder footer + welcome-sektionen för konsekvent layout.
Noterat att header-menyn kommer senare (när vi mergat header i main),  låter som en bra plan.
Från mitt håll ser detta redo ut att merge. Bra jobbat! 👏

leolinder-hash commented:
@VellaDona Hey! Tried to find the changes you suggested but I can't see that you made a review? I can only see your comment 🤔

leolinder-hash commented:
@VellaDona I've now updated the layout so it more resembles index.html. I've added a white background, changed the color of headlines and changed some buttons as well!

ReposByHussam commented:
All right, merging now!
